src/components/Link.tsx

```diff
@@ -82,7 +83,9 @@ export function useLink({
 
   const onPress = React.useCallback(
     (e: GestureResponderEvent) => {
-      outerOnPress?.(e)
+      const exitEarlyIfFalse = outerOnPress?.(e)
+
+      if (exitEarlyIfFalse === false) return
```

Comment:
Something I've been meaning to do, was gonna use for tags but going to do something else now. Might as well include.